exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0) {
+    return "negative";
+  } else {
```

Comment:
```suggestion
  } else {
```good job George, you can even delete the "else" here with the same result.